src/zmq/zmqpublishnotifier.cpp

```diff
@@ -227,50 +228,42 @@ bool CZMQPublishRawTransactionNotifier::NotifyTransaction(const CTransaction &tr
     return SendZmqMessage(MSG_RAWTX, &(*ss.begin()), ss.size());
 }
 
+// Helper function to send a 'sequence' topic message  with the following structure:
+//    <32-byte hash> | <1-byte label> | <8-byte LE sequence> (optional)
+static bool SendSequenceMsg(CZMQAbstractPublishNotifier& notifier, uint256 hash, char label, std::optional<uint64_t> sequence = {})
+{
+    unsigned char data[32+1+8];
+    for (unsigned int i = 0; i < 32; i++)
```

Comment:
even if its move only, I suggest adding brackets to confirm with the code style.

Reply:
s/i++/++i/

Reply:
Thanks, done (both suggested changes).